ocaml/libs/xapi-stdext/lib/xapi-stdext-threads/pthread_helpers.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) Cloud Software Group
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ */
+
+extern int pthread_set_name(const char *name);
```

Comment:
No needs for the `extern`.

Reply:
`pthread_` prefix is also used for system functions, I would avoid using the same prefix.